biogrid/libs/biogrid-simulator/src/lib/biogrid/biogrid.spec.ts

```diff
@@ -21,35 +22,39 @@ beforeAll(() => {
       energy: BUILDING.DEFAULT_INITIAL_ENERGY,
       x: 3,
       y: 4,
-      gridItemName: name1
+      gridItemName: name1,
     } as BuildingParams),
     new Building({
       energy: 0,
       x: 7,
       y: 9,
-      gridItemName: name2
+      gridItemName: name2,
     } as BuildingParams),
     new Building({
       energy: BUILDING.DEFAULT_INITIAL_ENERGY,
       x: 7,
       y: 8,
-      gridItemName: name3
+      gridItemName: name3,
     } as BuildingParams),
     new Building({
       energy: 0,
       x: 2,
       y: 1,
-      gridItemName: name4
+      gridItemName: name4,
     } as BuildingParams),
     new Building({
       energy: BUILDING.DEFAULT_INITIAL_ENERGY,
       x: 9,
       y: 9,
-      gridItemName: name5
+      gridItemName: name5,
     } as BuildingParams),
   ];
   grid = new Biogrid(
-    new RuralArea(ruralArea, /* townWidth = */ townWidth, /* townHeight = */ townHeight),
+    new RuralArea(
+      ruralArea,
```

Comment:
it's a bit confusing to have a RuralArea constructor take in something named ruralArea that is not itself a ruralArea object. 
Can we rename `ruralArea` object to be `buildingsInRuralArea` or `buildingList`